yarn.lock

```diff
@@ -0,0 +1,4207 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Here is a link for more details: https://auth0.com/blog/five-things-you-can-do-with-yarn/
It also helps us be consistent with our dependencies. It is basically `npm shrinkwrap`.